src/modules/pokemons/PokemonList.ts

```diff
@@ -21048,7 +21051,7 @@ export const pokemonList = createPokemonArray(
     },
     {
         'id': 800.03,
-        'name': 'Necrozma (Ultra)',
+        'name': 'Ultra Necrozma',
```

Comment:
Pokemon names are used as translation keys, so this will also need an update in the [translation repo](https://github.com/pokeclicker/pokeclicker-translations)

Reply:
Yep, I noticed that, already working on it.